### DISCUSSION TITLE
Infinite tower battle

### DISCUSSION THREAD

**Discussion by NoOneIsHereAgain** [Deijin27/RanseiLink#54] [Q&A]
So quick question, is there any way to edit the final battle with Nobunaga in the infinite tower?

And if not is that something you plan on adding in a future update?

**Comment by Deijin27** [comment]
I would really like to make it possible, but the data on these isn't stored in the same place as every other battle.

I've found it in none of the files so far, it's very likely hard-coded, thus requiring someone who knows how to reverse engineer game code, or at least find the data in there.

**Reply by NoOneIsHereAgain** [reply]
I see. Is that also the case for Hideyoshi in Viperia?

**Reply by Deijin27** [reply]
yeah
